src/Console.php

```diff
@@ -6,8 +6,16 @@
 
 use Closure;
 
-interface Console extends ConsoleInput, ConsoleOutput
+interface Console
 {
+    public function readln(): string;
```

Comment:
The idea here is that everything within `Console` interface is something users interact with directly. You'll see how it's written to the output later in the diff.